azure-pipelines.yml

```diff
@@ -47,8 +47,8 @@ jobs:
           conda config --set always_yes yes --set changeps1 no
           conda config --add channels omnia
           conda config --add channels janschulz
```

Comment:
Since the [janschulz channel no longer is maintained](https://anaconda.org/janschulz/repo) and we do not use these packages anymore, I think we can drop this channel completely.

Reply:
Removed janschulz